src/config/index.ts

```diff
@@ -12,12 +13,12 @@ const config = {
     'https://ipfs.everipedia.org/ipfs/',
   infuraId: process.env.NEXT_PUBLIC_INFURA_ID,
   alchemyApiKey: String(process.env.NEXT_PUBLIC_ALCHEMY_API_KEY),
-  alchemyChain: process.env.NEXT_PUBLIC_ALCHEMY_CHAIN || 'maticmum',
+  alchemyChain: process.env.NEXT_PUBLIC_ALCHEMY_CHAIN_POLYGON || 'maticmum',
```

Comment:
let's forget about matic for now. Let's focus in the IQ dashboard to connect with Ethereum mainnet in production and to goerli on dev.

I think its needed to modify the https://github.com/EveripediaNetwork/iq-ui/blob/main/src/config/wagmi.ts#L11

in order for using eth mainnet and goerli.

iqAddress: for ethereum mainnet is 0x579cea1889991f68acc35ff5c3dd0621ff29b0c9
for goerli no idea, probably @Emmanueldmlr might knw better

Reply:
@kesar, just to confirm, the IQ dashboard should only be connected to the Ethereum network right?

Reply:
it should connect to polygon as well in the future. But to simplify v1 we can focus in ethereum and extend it after launch

Reply:
Okay then. Thanks